proto-router/src/main/kotlin/actor/proto/router/ConsistentHashRouterState.kt

```diff
@@ -5,22 +5,25 @@ import actor.proto.PID
 import actor.proto.send
 import actor.proto.toShortString
 
+class ConsistentHashRouterRoutees (val hashRing: HashRing,val routeeMap:Map<String, PID>)
+
 internal class ConsistentHashRouterState(private val hash: (String) -> Int, private var replicaCount: Int) : RouterState() {
-    private lateinit var hashRing: HashRing
-    private lateinit var routeeMap: Map<String, PID>
+    private lateinit var routees : ConsistentHashRouterRoutees
```

Comment:
Replace the two different fields with one field containing an object with the two previous fields.'
This helps deal with the racyness of reading multiple fields.